workloads/whereabouts/run.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash -e
+
+set -e
+
+ES_SERVER=${ES_SERVER=https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com}
+LOG_LEVEL=${LOG_LEVEL:-info}
+if [ "$KUBE_BURNER_VERSION" = "default" ]; then
+    unset KUBE_BURNER_VERSION
+fi
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.1.0}
+CHURN=${CHURN:-false}
+WORKLOAD='node-density'
+QPS=${QPS:-20}
+BURST=${BURST:-20}
+GC=${GC:-true}
+EXTRA_FLAGS=${EXTRA_FLAGS:-}
+UUID=${UUID:-$(uuidgen)}
+KUBE_DIR=${KUBE_DIR:-/tmp}
+export ITERATIONS=${ITERATIONS:-341}
+
+download_binary(){
+  KUBE_BURNER_URL="https://github.com/kube-burner/kube-burner-ocp/releases/download/v${KUBE_BURNER_VERSION}/kube-burner-ocp-V${KUBE_BURNER_VERSION}-linux-x86_64.tar.gz"
+  curl --fail --retry 8 --retry-all-errors -sS -L "${KUBE_BURNER_URL}" | tar -xzC "${KUBE_DIR}/" kube-burner-ocp
+}
+function cleanup_whereabouts(){
+    
+    # remove IP pool
+    oc delete ippools.whereabouts.cni.cncf.io '10.1.0.0-21' -n openshift-multus
+
+    overlaps=$(oc get overlappingrangeipreservations.whereabouts.cni.cncf.io -n openshift-multus --no-headers=true | awk '{print $1}');
```

Comment:
shouldn't this be part of the exit criteria? After the workload completes and cleans up -- ie kube-burner deletes all the pods that it created, we should not have any IP addresses... if we do, we should exit 1 and have the status be that we leaked IPs, right?

Reply:
agreed. I added a global exit code for the overlaps, echo it, and then exit with a logical or between the overlaps and the job exit code.